.github/workflows/build-in-devcontainer.yaml

```diff
@@ -159,7 +159,11 @@ jobs:
             AWS_ACCESS_KEY_ID=${{ env.AWS_ACCESS_KEY_ID }}
             AWS_SESSION_TOKEN=${{ env.AWS_SESSION_TOKEN }}
             AWS_SECRET_ACCESS_KEY=${{ env.AWS_SECRET_ACCESS_KEY }}
-            RAPIDS_AUX_SECRET_1=${{ inputs.rapids-aux-secret-1 != '' && secrets[inputs.rapids-aux-secret-1] || '' }} # zizmor: ignore[overprovisioned-secrets]
+            RAPIDS_AUX_SECRET_1=${{
+              inputs.rapids-aux-secret-1 != ''
+              && secrets[inputs.rapids-aux-secret-1] # zizmor: ignore[overprovisioned-secrets]
+              || ''
+            }}
```

Comment:
The `env:` key for this action is a string field, not a YAML map.

The `# zizmor: ignore[overprovisioned-secrets]` comment here means the envvar is the literal value:
```
RAPIDS_AUX_SECRET_1="<secret> # zizmor: ignore[overprovisioned-secrets]"
```